app/presenters/edition_diff_presenter.rb

```diff
@@ -0,0 +1,24 @@
+module Presenters
+  class EditionDiffPresenter
+
+    def self.call(edition)
+      attributes = {}
+      return attributes unless edition.present?
+
+      Edition::TOP_LEVEL_FIELDS.each do |field|
+        attributes[field.to_s] = edition.public_send(field)
+      end
+
+      attributes["links"] = {}
+
+      edition&.links&.each do |link|
```

Comment:
Ah yes, I've realised in one of the tests an edition with no links was stubbed to return nil instead of an empty array, which was the reason I needed the safe navigation operator to make it work.  That's been change to return an empty array now 👍